modules/top-level/output.nix

```diff
@@ -140,7 +140,7 @@ with lib;
         '';
       };
 
-      extraConfigLuaPre = lib.optionalString config.wrapRc ''
+      extraConfigLuaPre = lib.mkIf config.wrapRc ''
```

Comment:
`extraPlugins` lower down in this file can also be a `mkIf`

Reply:
OK, I can change it too, but this PR is about cleaning up generated configs (mostly removing extra empty lines).

Reply:
When I read "treewide: optionalString -> mkIf" I didn't understand the motivation for the change and just assumed you were doing general cleanup.